spec/bookingsync/api/client/rentals_spec.rb

```diff
@@ -65,6 +65,16 @@
     it "returns a single rental" do
       rental = client.rental(prefetched_rental[:id])
       expect(rental.name).to eql(prefetched_rental[:name])
+      expect(rental).to_not have_key(:availability)
+    end
+
+    context "with additional query params" do
+      it "returns a single rental with defined params" do
+        rental = client.rental(prefetched_rental[:id], include: [:availability, :change_over])
+        expect(rental.name).to eql(prefetched_rental[:name])
```

Comment:
Is this `eql` intentional? I think we use `eq` everywhere, so ideally we should keep it consistent unless there are reasons for it. And `eql` is not the same as `eq`, check the following example:

```
 :006 > 1.0 == 1
 => true
 :007 > 1.0.eql?(1)
 => false
```